src/material-experimental/mdc-checkbox/checkbox.scss

```diff
@@ -74,6 +74,12 @@
     }
   }
 
+  // Clicking the label toggles the checkbox, but MDC does not include any styles that inform the
+  // user of this. Therefore we add the pointer cursor on top of MDC's styles.
+  label {
```

Comment:
Should this be reset for disabled checkboxes?

Reply:
Yep, good catch